lib/extension.js

```diff
@@ -116,6 +116,21 @@ Extension.prototype.callFunction = function (functionOptions, extensionOptions,
     return callback({statusCode: 422, errors: [errors.get('EXTENSION_INCORRECT_FUNCTION')]})
   }
 
+  if (extensionOptions.lastExportDateTime) {
+    try {
+      extensionOptions.lastExportDateTime = new Date(extensionOptions.lastExportDateTime)
+    } catch (err) {
+      return callback({statusCode: 422, errors: [{ message: err.message, code: err.code }]})
+    }
+  }
+  if (extensionOptions.currentExportDateTime) {
+    try {
+      extensionOptions.currentExportDateTime = new Date(extensionOptions.lastExportDateTime)
+    } catch (err) {
+      return callback({statusCode: 422, errors: [{ message: err.message, code: err.code }]})
```

Comment:
FYI, lastExportDateTime and currentExportDateTime should be taken from the functionOpts and I am working on PR for the same.